Modules/Core/Tests/Feature/ImportControllerTest.php

```
@@ -35,4 +37,368 @@ public function it_displays_import_page(): void
+        $files = $response->viewData('files');
+        
+        // Should only show allowed CSV files
+        $this->assertContains('clients.csv', $files);
+        $this->assertContains('invoices.csv', $files);
+        $this->assertNotContains('malicious.php', $files);
+        
+        // Cleanup
+        @unlink($uploadDir . '/clients.csv');
+        @unlink($uploadDir . '/invoices.csv');
+        @unlink($uploadDir . '/malicious.php');
+    }
```

PHPMD (2.15.0)

Remove error control operator '@' on line 164. (undefined)

(ErrorControlOperator)